stepaction/tekton-catalog-publish/0.1/tests/resources.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: catalog-source-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Mi
```

Comment:
do we need 500Mi for this test? 😅

Reply:
This was borrowed from the Task. For now I don't think so. Eventually StepActions will grow to the same size as Tasks. 

Actually, we could get by with an emptyDir here. Let me do that actually.